connector/src/test/java/org/apache/flink/connector/nebula/sink/AbstractNebulaOutPutFormatITTest.java

```diff
@@ -0,0 +1,175 @@
+package org.apache.flink.connector.nebula.sink;
+
+import java.util.concurrent.ExecutionException;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.TableEnvironment;
+import org.junit.Test;
+
+public class AbstractNebulaOutPutFormatITTest {
+    static final String META_ADDRESS = "127.0.0.1:9559";
+    static final String GRAPH_ADDRESS = "127.0.0.1:9669";
+    static final String USER_NAME = "root";
+    static final String PASSWORD = "nebula";
+
+    /** sink Nebula Graph Vertex Data with default INSERT mode */
+    @Test
+    public void sinkVertexData() throws ExecutionException, InterruptedException {
+        TableEnvironment tableEnvironment =
+                TableEnvironment.create(EnvironmentSettings.inStreamingMode());
```

Comment:
please make sure the ut works.
We can use docker-compose to start up nebula service and create a space before executing these tests.

Reply:
Fixed as below:
<img width="559" alt="image" src="https://user-images.githubusercontent.com/9884987/177292634-96635619-562f-48ae-ad23-cac4fc16b6d7.png">

Reply:
great tests